src/main/java/uk/gov/pay/adminusers/service/SecondFactorAuthenticator.java

```diff
@@ -1,31 +1,55 @@
 package uk.gov.pay.adminusers.service;
 
+import com.google.inject.Inject;
 import com.warrenstrange.googleauth.GoogleAuthenticator;
 import org.slf4j.Logger;
 import uk.gov.pay.adminusers.logger.PayLoggerFactory;
 
+import java.time.Clock;
+
 import static com.google.common.io.BaseEncoding.base32;
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 
 public class SecondFactorAuthenticator {
 
     private static final Logger logger = PayLoggerFactory.getLogger(SecondFactorAuthenticator.class);
+    private static final int VALID_PAST_TIME_WINDOWS = 4; // Plus 1 and then times 30 for maximum validity
```

Comment:
How about:
```
// To calculate maximum validity of an OTP token in seconds: ((VALID_PAST_TIME_WINDOWS + 1) * 30)
```

I'm just keen to keep this file well commented, I think it would be a complex thing to work out how to maintain without it.